src/js/generate-entries.sh

```diff
@@ -40,10 +40,6 @@ cd "$root"/tools/ts-generator/types &&
     --entities-define-file "$root"/src/js/modules/domain/generatedRpc/enableDisableCoproc.json \
     --output-file "$root"/src/js/modules/domain/generatedRpc/enableDisableCoprocClasses.ts
 
-cd "$root"/tools/ts-generator/rpc &&
```

Comment:
I think you might have to break this commit out into logical units or change the message based on the comment I left on the gen.json file